src/ex6/client/src/components/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "monday-ui-react-core/dist/main.css";
+import "../Header/Header.css";
+
+const Button = ({ children, ...props }) => {
+  switch (props.as) {
+    case "input":
+      return (
+        <input
+          type="checkbox"
+          name="done"
+          value="done"
+          id={props.id}
+          checked={props.status}
+          className="list-item-checkbox"
+          onChange={props.change}
+        >
+          {children}
+        </input>
+      );
+    case "button":
+      return (
+        <button onClick={props.click} type="button" id="list-item-submit">
+          +{children}
+        </button>
+      );
+    case "img":
+      return (
```

Comment:
The concept is cool. Maybe it would be nice to pass the content as children to the button.
for example
`<Button><img src= "" /> </Button>`

This way we can render whatever we like in the button while keeping it's core abilities